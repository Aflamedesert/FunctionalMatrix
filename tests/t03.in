X
10
2

