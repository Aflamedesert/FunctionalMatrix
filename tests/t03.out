Enter a character to fill the grid:
Enter rows and columns of X's:
X X 
X X 
X X 
X X 
X X 
X X 
X X 
X X 
X X 
X X 
